lib/shopify_app/scripttags_manager.rb

```diff
@@ -57,5 +60,11 @@ def scripttag_exists?(src)
     def current_scripttags
       @current_scripttags ||= ShopifyAPI::ScriptTag.all.index_by(&:src)
     end
+
+    def build_src(src)
+      src.respond_to?(:call) ? src.call(shop_domain) : src
+    rescue => e
+      raise CreationFailed, "cannot create script src: #{e.message}"
```

Comment:
My last commit does just that. May be hard to tell with this new :octocat: code review stuff but it's there. 

Also I've asked this 2 time already but here goes \#3: Under what circumstances [does this get called](https://github.com/Shopify/shopify_app/blob/076a2966823cfa2d74179cda675ff942de707c7a/lib/shopify_app/scripttags_manager.rb#L49)? If the API cannot create a resource an exception is raised.